build-system/status-page/incident.js

```diff
@@ -141,12 +141,12 @@ async function syncIncident() {
 
   // get step that was checked off
   let checked = -1;
-  for (const step of [3, 2, 1, 0]) {
+  for (const index of [3, 2, 1, 0]) {
```

Comment:
Same issue I had with the earlier remark about keeping this dynamic - this seems like a classic spot for an old-style for-loop: `for (let index = steps.length - 1; i >= 0; i--)`